classes/class-enqueues.php

```diff
@@ -0,0 +1,71 @@
+<?php
+/**
+ * Enqueue assets.
+ *
+ * @package Advanced_Multi_Block
+ */
+
+namespace Advanced_Multi_Block;
+
+/**
+ * Enqueues class.
+ */
+class Enqueues extends Plugin_Module {
+	/**
+	 * Path resolver for build directory.
+	 *
+	 * @var Plugin_Paths
+	 */
+	private Plugin_Paths $build_dir;
+
+	/**
+	 * Setup the class.
+	 *
+	 * @param string $build_path Absolute path to the build directory for all assets.
+	 */
+	public function __construct( string $build_path ) {
```

Comment:
This now receives the path of the `build` directory via constructor argument. It is now the responsibility of the caller of this class (plugin bootstrap) to define where that directory lives (which also gives us a single place to update it).